s64da_benchmark_toolkit/prepare.py

```diff
@@ -266,23 +266,24 @@ def add_indexes(self):
                 print(f'Applying {sql_file_path}')
                 with open(sql_file_path, 'r') as sql_file:
                     sql = sql_file.read()
-                    if sql:
-                        conn.cursor.execute(sql)
+
+                tasks = [self.psql_exec_cmd(cmd) for cmd in sql.split(';')]
```

Comment:
it would be safer to apply a sql parser to this? i assume the `;` is safe enough to split on but you never know.

Reply:
Point taken, there is [sqlparse](https://pypi.org/project/sqlparse/). Any objections from anybody else?